akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefSinkSpec.scala

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.scaladsl
+
+import akka.stream.ActorFlowMaterializer
+import akka.stream.testkit.AkkaSpec
+import akka.stream.testkit.StreamTestKit
+import akka.actor.Actor
+import akka.actor.ActorRef
+import akka.actor.Props
+
+object ActorRefSinkSpec {
+  case class Fw(ref: ActorRef) extends Actor {
+    def receive = {
+      case msg ⇒ ref forward msg
+    }
+  }
+}
+
+class ActorRefSinkSpec extends AkkaSpec {
+  import ActorRefSinkSpec._
+  implicit val mat = ActorFlowMaterializer()
+
+  "A ActorRefSink" must {
+
+    "send the elements to the ActorRef" in {
+      Source(List(1, 2, 3)).runWith(Sink.actorRef(testActor, onCompleteMessage = "done"))
```

Comment:
exactly what I was thinking about, then you have the full facilities of TestKit
@2m might be something to keep in mind when shaping up the stream-testkit